docs/development/bug-handling.md

```diff
@@ -0,0 +1,51 @@
+# Bug report handling
+
+## Start with a Discussion
+
+We use GitHub Discussions as a triaging stage for bug reports.
+Users should [create a new Request Help discussion](https://github.com/renovatebot/renovate/discussions/new?category=request-help) and choose from the options there.
+
+Bug reports often require additional information to resolve, so maintainers might ask directly for such information, or use automated label comments, or both.
+Bug reports which are missing the required additional information may be closed by maintainers if they feel that the Discussion as it exists is not beneficial to other users to leave open.
+
+We only create issues in this repository when we consider them to be actionable and work is ready to begin.
+Even if it seems highly likely that behavior is buggy, if there's no way to reproduce it or logs which can pinpoint it, then we usually won't create an Issue because it will just grow stale.
+
+## Discussion Resolution
+
+Bug reports can be resolved in one of three ways:
+
+1. Confirmed as a bug. The Discussion will be converted to an issue an issue and then closed, so that any future comments go to the Issue and not the Discussion.
+2. Rejected as "behaving as designed" with no further action needed. The Discussion will be closed with a relevant note.
+3. Rejected as not a bug, but accepted as a feature request. The Discussion will be closed with a note suggesting that the poster create a new "Suggest an Idea" Discussion instead, which can then be converted to an Issue once actionable.
+
+## FAQ
+
+### What's your definition of bug?
+
+A bug is classified as behavior in Renovate which is not as intended or documented.
+Missing functionality or incomplete support is not automatically a bug.
+For example, there's probably at least one feature of every package manager which is missing, and if we classified them all as "bugs" then the meaning of real bugs would be lost.
+
+### Why can't I create Bug Issues directly?
+
+Users have proven themselves to be poor judges of when something is a bug or not. Only a fraction of bug reports are actually bugs, and the remainder once used to pollute the repo issues, making it worse for maintainers and users alike to know what's "really" going on.
```

Comment:
```suggestion
It is really hard for users to decide if something is a bug.
Only a fraction of the incoming bug reports are for things we consider proper bugs.

A while ago, users were allowed to create Bug Issues.
We ended up with many "bug, but not really" type issues, that polluted the repository.
Those issues made it tricky for maintainers and users to see what's really going on in the repository.
We have since closed many old user-created bug Issues, and now only allow maintainers to create bug Issues.
This way we do not end up in the same situation again.
```

Simplify a bit, and add more context to the "history bit". And explain that because it was messy before, we now only create Issues ourselves.